### PR TITLE
Ensure that sizer is specified to have type int32.

### LINKED ISSUE #147: test_omnisci_column_basic crashes CUDA enabled omniscidb
The crash message:
```
NvidiaKernel.cpp:122 Check failed: cuLinkAddData_v2(link_state, CU_JIT_INPUT_PTX, static_cast<void*>(const_cast<char*>(ptx.c_str())), ptx.length() + 1, 0, 0, nullptr, nullptr) == CUDA_SUCCESS (218 == 0)
```

### PR DESCRIPTION
**Pull request by pearu**
Fixes #147.

This PR introduces a new type specifications `RowMultiplier` that is an alias to `int32|sizer=RowMultiplier`.

### CODE REVIEW

File: rbc/tests/test_omnisci_column_basic.py [hunk 1]
```diff
@@ -223,14 +206,14 @@ def add_columns(x, y, alpha, m, r):
...
-                    ' [rbc issue 147]')
+                    ' [rbc issue 171]')
     omnisci.reset()
     # register an empty set of UDFs in order to avoid unregistering
     # UDFs created directly from LLVM IR strings when executing SQL
     # queries:
     omnisci.register()
 
-    @omnisci('int32(Column<double>, int64|sizer=RowMultiplier,'
+    @omnisci('int32(Column<double>, RowMultiplier,'
```
Comment (guilhermeleobas) [inline rbc/tests/test_omnisci_column_basic.py:216]: RBC will automatically convert `RowMultiplier` to `int32|sizer=RowMultiplier`?

Comment (pearu) [reply rbc/tests/test_omnisci_column_basic.py:216]: No. There will be an exception raised, see
https://github.com/xnd-project/rbc/pull/170/files#diff-8e1cf1ddf04a5e6c92ef9bb819f2a10dR627

**Comment by pearu** [comment]
> LGTM. One question, if the user specifies a `int64=sizer|...`, RBC will convert sizer to an int32?

No. There will be an exception raised, see
https://github.com/xnd-project/rbc/pull/170/files#diff-8e1cf1ddf04a5e6c92ef9bb819f2a10dR627

File: rbc/omniscidb.py [hunk 2]
```diff
@@ -621,6 +623,8 @@ def is_udtf(sig):
                             annot = a.annotation()
                             _sizer = annot.get('sizer', unspecified)
                             if _sizer is not unspecified:
+                                # sizer must have type int32
+                                assert a.is_int and a.bits == 32, a
```
Comment (guilhermeleobas) [inline rbc/omniscidb.py:627]: So, maybe add an error message `sizer must have type int32` in case the assertion fails.

**Review by guilhermeleobas — APPROVED** [review]

**Comment by pearu** [comment]
Requires https://github.com/omnisci/omniscidb-internal/pull/4785

**Pull request merged**